doc/specs/vulkan/appendices/VK_NV_clip_space_w_scaling.txt

```diff
@@ -18,27 +18,28 @@ to the final post-processed image.
 This extension provides a mechanism to render VR scenes at a non-uniform
 resolution, in particular a resolution that falls linearly from the center
 towards the edges.
-This is achieved by scaling the "w" coordinate of the vertices in the clip
-space before perspective divide.
-The clip space "w" coordinate of the vertices can: be offset as of a
-function of "x" and "y" coordinates as follows:
+This is achieved by scaling the [eq]#w# coordinate of the vertices in the
```

Comment:
I'm not sure why you're converting plaintext "w" into [eq]#w# here, while converting [eq]#s# into *s* below. For single variables it makes little difference, but consistency is nice.

I like having the [eq] style tag on equations in order to keep track of them, but haven't yet customized a stylesheet to do anything special. Eventually they should use the same font and italicization conventions as latexmath markup does.

Reply:
If it appears in an equation somewhere I make it `[eq]` too.
If it doesn't I make it bold with `*`, which is more noticable.
I think elsewhere @TobiasHector advised me to sometimes use `*var*`.

Reply:
PS: not sure if a style can help things. Mostly we would probably want only the names in italic, while the rest (e.g. operators + -  , or function names) straight.